lib/active_merchant/billing/gateways/stripe.rb

```diff
@@ -311,6 +311,8 @@ def create_post_for_auth_or_purchase(money, payment, options)
           post[:receipt_email] = options[:receipt_email] if options[:receipt_email]
           add_customer(post, payment, options)
           add_flags(post, options)
+        else
```

Comment:
Optional: I think this would read a bit easier now as:
```
if emv_payment?(payment)
  add_statement_address(post, options)
else
  add_amount(post, money, options, true)
  add_customer_data(post, options)
  post[:description] = options[:description]
  post[:statement_descriptor] = options[:statement_description]
  post[:receipt_email] = options[:receipt_email] if options[:receipt_email]
  add_customer(post, payment, options)
  add_flags(post, options)
end
```